commons-profiler-api/src/main/java/ru/fix/commons/profiler/impl/SharedCounters.java

```diff
@@ -57,4 +78,54 @@ public MaxThroughputCalculator getMaxThroughput() {
     public MaxThroughputCalculator getMaxPayloadThroughput() {
         return maxPayloadThroughput;
     }
+
+    public LongAdder getActiveCallsCounter() {
+        return activeCallsCounter;
+    }
+
+    public ActiveCallsSet getActiveCalls() {
+        return activeCalls;
+    }
+
+    public class ActiveCallsSet implements Iterable<ProfiledCallImpl> {
+        private ConcurrentHashMap<ProfiledCallImpl, Boolean> activeCalls = new ConcurrentHashMap<>();
+
+        public void add(ProfiledCallImpl call) {
+            if (recordActiveCalls.get()) {
+                activeCalls.put(call, true);
+            }
+        }
+
+        public void remove(ProfiledCallImpl call) {
+            activeCalls.remove(call);
+        }
+
+        public boolean isEmpty() {
+            return activeCalls.isEmpty();
+        }
+
+        public int size() {
+            return activeCalls.size();
+        }
+
+        public boolean containsAll(Collection<?> collection) {
+            return activeCalls.keySet().containsAll(collection);
+        }
+
+        @Override
+        public Iterator<ProfiledCallImpl> iterator() {
+            return activeCalls.keySet().iterator();
+        }
+
+        public Stream<ProfiledCallImpl> stream() {
+            return activeCalls.keySet().stream();
+        }
+
+        /**
+         * NOT SYNCHRONIZED!
+         */
+        public void reset() {
+            activeCalls = new ConcurrentHashMap<>();
```

Comment:
Could you please clarify this `NOT SYNCHRONIZED` block? 
Looked like a concurrent bug and should be probably replaced by `activeCalls.clear();`